File: agent-driven-architecture-main/examples/typescript/test/gate/fixtures/violating/C17/src/blocks/escalation/contract.ts
// The DECLARATION the census derives from — identical in shape to the live
// contract, so the derivation cannot be satisfied by a fixture idiom the tree
// migrated away from (the C7 rot). The literal type positions here are MATCHES
// and are never counted as constructions.
import type { EffectBase } from "@adr/spine/pure/effect";

export interface PageOncall extends EffectBase {
  readonly kind: "PageOncall";
  readonly effectClass: "Irreversible";
  readonly ticket: string;
}

File: agent-driven-architecture-main/examples/kotlin/src/test/fixtures/konsist/violating/C17/blocks/escalation/Contract.kt
// The DECLARATION the census derives from — identical in shape to the live contract,
// so the derivation cannot be satisfied by a fixture idiom the tree migrated away
// from (the C7 rot). The superclass CALL is what classifies the leaf.

package adr.contract

import adr.spine.pure.EffectClass
import adr.spine.pure.TicketId
import adr.spine.pure.Timestamp

sealed class EscalationEffect(
    override val at: Timestamp,
    effectClass: EffectClass,
) : Effect(at, effectClass) {
    data class PageOncall(
        override val at: Timestamp,
        val ticket: TicketId,
    ) : EscalationEffect(at, EffectClass.Irreversible)
}
